DotNetCliPerf/CoreApp.cs

```diff
@@ -23,6 +23,12 @@ public abstract class CoreApp : DotNetApp
         [Params(MSBuildFlavor.Framework, MSBuildFlavor.Core)]
         public MSBuildFlavor MSBuildFlavor { get; set; }
 
+        [Params("2.0", "2.1")]
+        public string CoreTargetFramework { get; set; }
+
+        [Params(false, true)]
+        public bool UseBuildServer { get; set; }
```

Comment:
This should probably say `UseRazorBuildServer`

Reply:
Looking at the usage - does this switch really mean `RazorCompileOnBuild`?

We're not really interested in how much the server affects the performance, we're interested in how much compiling Razor at build time affects the performance. We don't care about the results of using Razor at build time without the server, we already know those numbers are bad.

Reply:
> does this switch really mean RazorCompileOnBuild?

Yes it means "Compile on build with the build server". I'll change the name.